_stbt/core.py

```diff
@@ -1003,8 +1003,17 @@ def _callable_description(callable_):
     >>> _callable_description(
     ...     lambda: stbt.press("OK"))
     '    lambda: stbt.press("OK"))\\n'
+    >>> _callable_description(functools.partial(int, base=2))
+    'int'
     """
-    d = callable_.__name__
+    try:
+        d = callable_.__name__
+    except AttributeError:
+        if isinstance(callable_, functools.partial):
+            # functools.partial wraps the original function
+            d = callable_.func.__name__
```

Comment:
Perhaps it would be worth including the baked-in arguments? Something like:
```py
name = callable_.func.__name__
args = ', '.join(str(arg) for arg in callable_.func.args)
kwargs = ', '.join('{}={!r}'.format(arg, value) for arg, value in callable_.func.keywords.iteritems())
d = '{}({}, {})'.format(name, args, kwargs)
```
(That doesn't handle the case where `args` or `kwargs` is empty, but something like that...)